src/utils/hooks/useScroll.ts

```diff
@@ -0,0 +1,26 @@
+import { useEffect } from 'react';
+
+const top: ScrollToOptions = {
+  top: 0,
+  left: 0,
+  behavior: 'smooth'
+};
+
+export const useScroll = (
+  dispatcher: any,
```

Comment:
rename dispatcher to trigger this is a little bit confusing. Also add genertic type - any will be problem after merging new es lint rules